R/extremes-data.R

```diff
@@ -7,6 +7,8 @@
 extremesTable <- function(rawData){
   
   data <- applyQualifiers(rawData)
+  l_qual <- data$dv$qualifiers
+  if (length(l_qual)==0) return ("The dataset requested is empty.")
```

Comment:
The data does not have to have qualifiers, this is cutting off rendering of valid data sets.